src/apiCalls.js

```diff
@@ -0,0 +1,7 @@
+function fetchCustomerData() {
+  return fetch('http://localhost:3001/api/v1/customers')
+  .then(response => response.json())
+  .catch(err => showError(err));
+}
+
+export{fetchCustomerData}
```

Comment:
Looks good! You could also start your function off as "export const" fetchCustomerDate = () => {} if you need to adjust it.